### DISCUSSION TITLE
More basic facts on clues for constructors (pre-solve)

### DISCUSSION THREAD

**Discussion by revuluri** [crosshare-org/crosshare#492] [Ideas]
As a constructor, I would like to be able to see some basic information about my **clues** as well as entries (before publishing or any solving). (The basic information on the fill/entries, including the number of words, mean word length and distribution, number and percentage of blocks, and letter counts are very helpful.) A few different ideas:

- One thing in particular would be the total character count of all the clues together.
- One aspiration would be some measure, even if rough, of ["freshness"](https://www.xwordinfo.com/Fresh) for the clues (I know that this does not yet exist for the grid either).
- One other thing that might be useful is a sense of where these measures live in the distribution (e.g., percentile) for all puzzles of the same size (either exactly the same dimensions, or roughly the same size like within 5% of total square count in the grid). Some of the inspiration for this comes from the [Crosserville Stats page](https://www.crosserville.com/stats).
- Eventually it might be nice to aggregate across all puzzles I have constructed (or all puzzles on Crosshare) to see a mean and/or distribution/percentile. 

I don't have a clear sense of where in the workflow these would best live. Happy to think more if that's helpful!

**Comment by mdirolf** [comment]
I love the idea to add more pre-solve facts / stats. I'm going to need some time to think through what is feasible in the short / medium term but I think it makes sense to then open individual issues for discrete units of work. In the meantime let's use this discussion to add any more thoughts / ideas!
